vendor/mozilla-rust-sdk/googleapis/src/spanner.rs

```diff
@@ -36,7 +49,7 @@ impl Client {
         let creds = ChannelCredentials::google_default_credentials()?;
 
         // Create a Spanner client.
-        let chan = ChannelBuilder::new(env)
+        let chan = ChannelBuilder::new(env.clone())
```

Comment:
Does clippy complain if we don't have this `clone()` here?

Reply:
If it's now needed env is an `Arc` so this would be preferred:

```suggestion
        let chan = ChannelBuilder::new(Arc::clone(&env))
```

Reply:
This probably should be corrected in the mozilla-rust-sdk.